### PR TITLE
Address

### PR DESCRIPTION
**Pull request by lamerritt1**
Updated regex and test cases to run with no errors

### CODE REVIEW

**Review by claesmk — APPROVED** [review]
Looks good

**Comment by claesmk** [comment]
Reminder @lamerritt1 you are waiting on @TangBaiFeng to review your code before you will be able to merge.  You can also pick someone else on the team if you like

**Review by TangBaiFeng — APPROVED** [review]

**Pull request merged**
